backend/EasyMeets.Core/EasyMeets.Core.DAL/Entities/Calendar.cs

```diff
@@ -0,0 +1,12 @@
+namespace EasyMeets.Core.DAL.Entities;
+
+public class Calendar : AuditEntity<int>
+{
+    public int? AddEventsFromTeamId { get; set; }
+    public bool CheckForConflicts { get; set; }
+    
+    public User User { get; set; }
+    public Team? ImportEventsFromTeam { get; set; }
```

Comment:
Don't we need Ids here?

Reply:
UserId added
AddEventsFromTeamId was already exist